hotness/consumers.py

```diff
@@ -698,46 +412,50 @@ def is_monitored(self, package):
             return False
 
         try:
-            data = r.json()
-            package = data['packages'][0]
-
-            # Even if the package says it is monitored.. if it is retired, then
-            # let's not file any bugs or anything for it.
-            if package['package']['status'] == "Retired":
-                return False
-
-            # Otherwise, if it is not retired, then return the monitor flag.
-            return package['package'].get('monitor', True)
+            data = yaml.safe_load(r.text)
+            string_value = data.get('monitoring', 'no-monitoring')
+            return_values = {
+                'monitoring': 'nobuild',
+                'monitoring-with-scratch': True,
+                'no-monitoring': False,
+            }
+            return return_values[string_value]
         except:
-            _log.exception("Problem interacting with pkgdb.")
+            _log.exception("Problem interacting with the pagure repo.")
             return False
 
+    @logged
+    def is_retired(self, package):
+        url = '{0}/rest_api/v1/component-branches/'.format(self.pdc_url)
+        params = dict(
+            name='master',
+            global_component=package,
+            type='rpm',
+            active=True,
+        )
+        _log.debug("Checking %r, %r" % (url, params))
```

Comment:
It'd be nice if this was more informative. Something like
```
'Querying {url} to determine if {package} is retired'
```
And then some sort of followup with the result.

Alternatively, you could make the request first and log in a single statement the result.

Reply:
Added some more context in 92eeeb3.